dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/McpServlet.java

```diff
@@ -204,4 +229,22 @@ private void ping(McpTransport transport) {
         transport.sendResponse(new Object());
     }
 
+    private void cancelRequest(McpTransport transport) {
+        McpNotificationParams notificationParams = transport.getMcpRequest().getParams(McpNotificationParams.class, jsonb);
+        RequestId requestId = new RequestId(notificationParams.getRequestId(), transport.getRequestIpAddress());
+        Optional<String> reason = Optional.ofNullable(notificationParams.getReason());
+
+        Cancellation cancellation = connection.getOngoingRequestCancelation(requestId);
+
+        if (cancellation != null) {
+            ((CancellationImpl) cancellation).cancel(reason);
+            transport.sendEmptyResponse();
+        }
+        //Ignore notification if there is no request to cancel
```

Comment:
```suggestion
        if (cancellation != null) {
            ((CancellationImpl) cancellation).cancel(reason);
        }
        //Ignore notification if there is no request to cancel
        transport.sendEmptyResponse();
```

I think we want to send an empty response regardless of whether there was anything to cancel.

Reply:
Done